.github/workflows/cd.yml

```diff
@@ -3,6 +3,8 @@ on:
   push:
     branches: [main]
 
+permissions: write-all
```

Comment:
We used default-access permissive (see https://docs.github.com/en/actions/security-guides/automatic-token-authentication)

This add metadata +w and id-token +rw